test/Microsoft.NET.Publish.Tests/GivenThatWeWantToPreserveCompilationContext.cs

```diff
@@ -35,6 +35,7 @@ public void It_publishes_the_project_with_a_refs_folder_and_correct_deps_file()
                     propertyGroup.Should().NotBeNull();
 
                     propertyGroup.Add(new XElement(ns + "DisableImplicitFrameworkReferences", "true"));
+                    propertyGroup.Add(new XElement(ns + "DisableLockFileFrameworks", "true"));
```

Comment:
Thinking about this a bit more, I think that even if Microsoft/msbuild#1345 is fixed, that it could still be a problem if there is a reference to the same framework assembly coming both from a NuGet FrameworkReference and the implicit framework references from the SDK.

So can you either disable the implicit framework references if targeting .NET 4.5 or greater (which is the lowest thing supported by .NET Standard), or file a bug to follow up and investigate?